### ISSUE TITLE
Widget

### ISSUE THREAD

**Issue by ZaldyColinares** [25huizengek1/ViTune#348]
## Feature request: ...

We would love to see a material you style widget integrated to ViTune, the predecessor would be so proud 🥲

**Comment by 25huizengek1** [comment]
I guess we could give it another shot (#27)

**Comment by ZaldyColinares** [comment]
> I guess we could give it another shot (#27)

Lovely 🌹

**Comment by OutshineIssue** [comment]
Lol I was really exited when I saw this one completed, probably a mistake? 😆

**Comment by ZaldyColinares** [comment]
> Lol I was really excited when I saw this one completed, probably a mistake? 😆

Nope, I just reopened it maybe because the dev might post updates here xD

**Comment by ZaldyColinares** [comment]
Hi, I hope you're well, I want to know the progress or when the widget arrive? :3

**Comment by Kimiberlianoputra** [comment]
Hm... not yet?

**Comment by Wqttzicue** [comment]
> Hm... not yet?

Yes, this suggestion is about 4 months on Low priority

**Comment by Kimiberlianoputra** [comment]
> > Hm... not yet?
> 
> Yes, this suggestion is about 4 months on Low priority

😭
